SpotifyArtistsViewer/SARequestManager.swift

```diff
@@ -0,0 +1,101 @@
+//
+//  SARequestManager.swift
+//  SpotifyArtistsViewer
+//
+//  Created by Kevin Zeckser on 6/6/16.
+//  Copyright © 2016 Kevin Zeckser. All rights reserved.
+//
+import Foundation
+
+enum SAResponse {
+    case Success(artists: [SAArtist])
+    case Failure(error: ErrorType)
+}
+
+
+final class SARequestManager {
+    
+    // MARK: - Properties
+    
+    static let sharedInstance = SARequestManager()
+    private let session: NSURLSession!
+    // This prevents others from using the default '()' initializer for this class.
+    private init() {
+        let sessionConfig: NSURLSessionConfiguration = NSURLSessionConfiguration.defaultSessionConfiguration()
```

Comment:
you don't need the ": NSURL... " type on the lefthand side here, as it's already inferred from the right side.